.github/workflows/_build_iwasm_release.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: build iwasm release
+
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: arch of the release
+        type: string
+        required: false
+        default: x86_64
+      cwd:
+        description: workfing directory
+        type: string
+        required: true
+      runner:
+        description: OS of compilation
+        type: string
+        required: true
+      upload_url:
+        description: a semantic version number. it is required when `release` is true.
+        type: string
+        required: false
+      ver_num:
+        description: a semantic version number. it is required when `release` is true.
+        type: string
+        required: false
+
+jobs:
+  build:
+    runs-on: ${{ inputs.runner }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: generate iwasm binary release
+        run: |
+          cmake -S . -B build \
+            -DWAMR_BUILD_AOT=1 -DWAMR_BUILD_INTERP=1 -DWAMR_BUILD_JIT=0 \
+            -DWAMR_BUILD_CUSTOM_NAME_SECTION=0 \
+            -DWAMR_BUILD_DEBUG_INTERP=0 \
+            -DWAMR_BUILD_DEBUG_AOT=0 \
+            -DWAMR_BUILD_DUMP_CALL_STACK=0 \
+            -DWAMR_BUILD_LIBC_BUILTIN=0 \
+            -DWAMR_BUILD_LIBC_UVWASI=0 \
```

Comment:
For Windows, WAMR_BUILD_LIBC_UVWASI should be 1, since wasmtime wasi implemented cannot be built in Windows.

And WAMR_BUILD_LIBC_WASI isn't set, had better set it to 1 explicitly.